_infra/helm/securebanking-openbanking-uk-rs/templates/deployment.yaml

```diff
@@ -77,7 +77,7 @@ spec:
         {{- if .Values.springConfig }}
           - name: spring-config
             configMap:
-              name: {{ .Chart.Name }}-spring-config
+              name: application-config
```

Comment:
should this be test-facility-bank-app-config? Does this config only relate to the rs?

Reply:
This is the reference related with the spring config map, created in the configmap descriptor.
From slack channel posted by @shaunharrisonFR:
```
yeah likely need to review them at some point but fine for now
as in whats in the configmaps
so securebanking-openbanking-uk-rs-spring-config
this is created within RS
name: {{ .Chart.Name }}-spring-config
so that can simply be changes to name: application-config
```